pkg/apis/migration/v1alpha1/directvolumemigration_types.go

```diff
@@ -27,7 +27,8 @@ import (
 type PVCToMigrate struct {
 	*kapi.ObjectReference `json:",inline"`
 	TargetStorageClass    string                            `json:"targetStorageClass"`
-	TargetAccessModes     []kapi.PersistentVolumeAccessMode `json:"targetAccessModes,omitEmpty"`
+	TargetAccessModes     []kapi.PersistentVolumeAccessMode `json:"targetAccessModes,omitempty"`
+	TargetNamespace       string                            `json:"targetNamespace,omitempty"`
 	Verify                bool                              `json:"verify,omitEmpty"`
```

Comment:
```suggestion
	Verify                bool                              `json:"verify,omitempty"`
```

Reply:
Ahh, good catch. I noticed the problem with AccessModes but it's here for Verify as well.